src/Adapters/Email/SimpleMailgunAdapter.js

```diff
@@ -0,0 +1,44 @@
+import Mailgun from 'mailgun-js';
+
+let SimpleMailgunAdapter = mailgunOptions => {
+	if (!mailgunOptions || !mailgunOptions.apiKey || !mailgunOptions.domain) {
+		throw 'SimpleMailgunAdapter requires an API Key and domain.';
+	}
+	let mailgun = Mailgun(mailgunOptions);
+
+	let sendMail = ({to, subject, text}) => {
+		let data = {
+			from: mailgunOptions.fromAddress,
+			to: to,
+			subject: subject,
+			text: text,
+		}
+
+		return new Promise((resolve, reject) => {
+			mailgun.messages().send(data, (err, body) => {
+				if (typeof err !== 'undefined') {
+					reject(err);
+				}
+				resolve(body);
+			});
+		});
+	}
+
+	return Object.freeze({
+		sendVerificationEmail: ({ link, user, appName, }) => {
+			let verifyMessage =
+	      "Hi,\n\n" +
+	      "You are being asked to confirm the e-mail address " + user.email + " with " + appName + "\n\n" +
+	      "" +
+	      "Click here to confirm it:\n" + link;
+			return sendMail({ 
+        to:user.email, 
```

Comment:
My standing is that for places where we want to allow CRUD, like in Parse.Cloud things - it makes an absolute sense to do this.
Otherwise, where this is not the intention of the adapter - you can still do it by creating a Parse.Object from JSON object that you get in the adapter, but it's in your best interest to actually not do it.